containers/anndata-to-ui/create-test-input.py

```diff
@@ -38,10 +38,12 @@ def create_h5ad_secondary_analysis(h5ad_path):
 
 
 def create_h5ad_scvelo(h5ad_path):
-    data = array(
+    # scvelo uses spliced inputs:
+    # https://github.com/hubmapconsortium/salmon-rnaseq/blob/master/bin/analysis/scvelo_analysis.py#L13 
+    data = csr_matrix(array(
         [[i for i in range(15)], [i for i in range(15)], [i for i in range(15)]]
-    )
-    layers={ 'spliced_unspliced_sum': csr_matrix(data) }
+    ))
+    layers={ 'spliced': data }
```

Comment:
I'm not sure whether it's still necessary to save a separate `spliced` key now that no log-scaling or normalization need to be performed in this code, but I don't have a stake in that either.

Reply:
I think I will leave it for now.  Eventually we'll actually make use of the layers once `AnnData` gives us the degree of control needed.